### ISSUE TITLE
Armour Chest Feature does not render trims

### ISSUE THREAD

**Issue by mapleexists** [exaskye/Ears#141]
When a skin has the Chest feature enabled, wearing a chestplate with trims does not display the trim on the extra chest part:

![2023-09-26_15 19 29](https://github.com/unascribed/Ears/assets/27739930/44e540c8-0f0b-4a85-b3dc-e5fccbbe91c2)

Somewhat related to #103

**Comment by exaskye** [comment]
Yes, the 1.20 ports were very lazy as Ears has become unwieldy and unmaintainable, and is in the middle of a rewrite. There is no ETA on the rewrite.

**Comment by Antikyth** [comment]
<img width="3440" height="1351" alt="Image" src="https://github.com/user-attachments/assets/dcaf1d44-0b0b-4a64-a253-0fc389e4a2ea" />

Disclaimer: I do have Sodium installed.

As of Fabric 1.21.8, not only does the chest render through the armor, but where the chestplate should be rendered on top it seems some kind of armor trim texture atlas is rendered for the texture.
